### PR TITLE
WPCOM Plugins: Add class names and style file

### PR DESCRIPTION
**Pull request by dmsnell**
This patch introduces some basic class names on the wpcom plugin panel
area and introduces a style file to contain the necessary visual
adjustments.

This area is still hidden behind a feature flag and is only present on
development builds.

**Testing**

Navigate to
http://calypso.localhost:3000/plugins/testblogdomain.wordpress.com

You should see the unstyled panels.

screen shot 2016-03-23 at 5 42 23 pm

cc: @drw158 - I'm hoping this will help you get started to see the structure, make some styling, and give me feedback on changes that would help you out

### CODE REVIEW

File: client/my-sites/plugins-wpcom/info-header.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Card from 'components/card';
 export const InfoHeader = React.createClass( {
 	render() {
 		return (
-			<Card>
+			<Card className="wpcom-plugins-info-header">
```
Comment (mtias) [inline client/my-sites/plugins-wpcom/info-header.jsx]: In this case, with `info-header` being a sub-component of `plugins-wpcom`, the class should be more like `.plugins-wpcom__info-header`. Any class attribute within the `plugins-wpcom` folder should have the folder as the prefix.

File: client/my-sites/plugins-wpcom/plugin-types/premium-plugin.jsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const PremiumPlugin = React.createClass( {
 		} = this.props;
 
 		return (
-			<div>
+			<div className="wpcom-premium-plugin">
```
Comment (mtias) [inline client/my-sites/plugins-wpcom/plugin-types/premium-plugin.jsx]: Here same as above, but with `plugin-types`, whose name could maybe be improved? Maybe `wpcom-plugin-item`?

File: client/my-sites/plugins-wpcom/style.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.wpcom-stanard-plugins-panel__action-button {
```
Comment (mtias) [inline client/my-sites/plugins-wpcom/style.scss]: Small typo in `stanard`.

File: client/my-sites/plugins-wpcom/info-header.jsx (hunk 1 above)
Comment (dmsnell) [inline client/my-sites/plugins-wpcom/info-header.jsx]: Thanks for the tip. I had originally thought about that and then changed my mind to try and keep the components independent. Will update to the "dependency" though.

**Comment by dmsnell** [comment]
@mtias some updates for you - thanks again for your feedback!

**Comment by mtias** [comment]
Thank you! This is looking good to me, just a nitpick that we have the folder as `plugins-wpcom` and the class as `wpcom-plugins`.

Depending on how the design shapes up to be I also think we may have an opportunity to merge the `wpcom-plugin-item` component with the one that has been created for the thank-you page down the line:

image

**Comment by dmsnell** [comment]
> merge the wpcom-plugin-item component

that looks nice. is it sitting in another branch at the moment?

**Comment by mtias** [comment]
I think it's the <a href="https://github.com/Automattic/wp-calypso/tree/master/client/components/purchase-detail">PurchaseDetail</a> component (which we should move to "client/my-sites" and app components in devdocs and probably rename to "feature" or something similar).

**Comment by drw158** [comment]
:+1:

**Comment by dmsnell** [comment]
@mtias I will earmark that component for the future. for now I'm going to go ahead and merge so @drw158 can get started doing his magic. This code right now won't even be in production, so still easy to change.

**Comment by mtias** [comment]
Sure thing, it's just something we should look at aligning down the road if it makes sense.

**Pull request merged**